### PR TITLE
Release 4.1.2

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.1.2 (2020-07-13)
+
+- Fixed the date pipe to allow for data which is not a `Date` object. [#186](https://github.com/blackbaud/skyux-datetime/pull/186)  (Thanks [@Blackbaud-NickGlyder](https://github.com/Blackbaud-NickGlyder)!)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: Does it mean the same thing to say, "Fixed the date pipe to allow `any` as the value type." ?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.1.2](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.1.2)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md:3]: Ya. To me that reads a little funny though as the `any` value type just means that you can pass whatever value type in that you want. So we aren't really "allowing the `any` type" as that is just allowing all types.

Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md:3]: Fair point! I looked at the associated PR and was curious if it was worth calling out that way.

Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Would it make sense to make change this to something like `Fixed the date pipe to allow all data types.`?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done. I also updated the date.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.1.2](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.1.2)

**Pull request merged**